WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -574,7 +575,14 @@ public static String getLoginUrl(Blog blog) {
     public static String getDefaultUserAgent() {
         if (mDefaultUserAgent == null) {
             // TODO: use WebSettings.getDefaultUserAgent() after upgrade to API level 17+
-            mDefaultUserAgent = new WebView(getContext()).getSettings().getUserAgentString();
+            try {
+                // Catch AndroidRuntimeException that could be raised by the WebView() constructor.
+                // See https://github.com/wordpress-mobile/WordPress-Android/issues/3594
+                mDefaultUserAgent = new WebView(getContext()).getSettings().getUserAgentString();
```

Comment:
That SO poster says they only saw this on 5.0 devices, so would it be worthwhile using `WebSettings.getDefaultUserAgent()` to avoid the crash when we detect API 17+?